src/anki/util/Functions.h

```diff
@@ -151,13 +151,36 @@ inline T max(T a, T b)
 	return (a > b) ? a : b;
 }
 
+/// Returns 1 or -1 based on the sign
+template <typename T>
+inline T sign(T v) 
```

Comment:
maybe you can move those functions (sign, wrap, smoothstep) **including** clamp to math/Functions.h. These feel more of math type functions than generic ones.

Reply:
That's the first feeling I got but then I saw clamp being defined there. I'll move the functions